bdd_tests/features/steps/database.py

```diff
@@ -232,6 +232,8 @@ def insert_report_with_risk_in_new_reports_table(context, risk):
     else:
         raise ValueError(f"Invalid category of total risk {risk}. Expected one of ['low', 'moderate', 'important', 'critical']")
 
+    if not updated_at:
```

Comment:
can be added into line #222 directly

Reply:
It would always evaluate to the same timestamp, and that would lead to unique key violation when inserting (e.g `DETAIL:  Key (org_id, cluster, updated_at)=(1, 5d5892d4-2g85-4ccf-02bg-548dfc9767aa, 2021-12-22 09:59:24.907496) already exists.`)

Reply:
oh, that's true. The datetime.now() is called in the creation of the function, not every time you call it. Nice catch